### ISSUE TITLE
BUG: Model Files do not allow spaces

### ISSUE THREAD

**Issue by ShadowLordAlpha** [sriharshachilakapati/SilenceEngine#20]
`Model.load("models/character test.obj");` throws an exception and only has the `models/character` in the error string and not the full path. There is also no way to get around this like in other areas where an `InputStram` can be passed in. Directly calling the loader could be done but the formatting for the string the loader takes is still undocumented

**Comment by ShadowLordAlpha** [comment]
This also affects the mtl file

**Comment by sriharshachilakapati** [comment]
Can you please post the stacktrace of the exception? That should be helpful in finding the source of the error.

**Comment by ShadowLordAlpha** [comment]
Stacktrace
```Java
com.shc.silenceengine.core.SilenceException: Resource not found: models/Character Test.obj
	at com.shc.silenceengine.utils.FileUtils.getResource(FileUtils.java:81)
	at com.shc.silenceengine.models.obj.OBJModel.parseOBJ(OBJModel.java:72)
	at com.shc.silenceengine.models.obj.OBJModel.<init>(OBJModel.java:65)
	at com.shc.silenceengine.models.Model.load(Model.java:40)
	at yirath.state.PlayState.onEnter(PlayState.java:41)
	at com.shc.silenceengine.core.Game.setGameState(Game.java:155)
	at yirath.state.MenuState$2.invoke(MenuState.java:79)
	at com.shc.silenceengine.core.glfw.Window.lambda$6(Window.java:462)
	at com.shc.silenceengine.core.glfw.Window$$Lambda$26/1347137144.invoke(Unknown Source)
	at org.lwjgl.glfw.GLFW$10.invoke(GLFW.java:2841)
	at org.lwjgl.glfw.GLFWKeyCallback.callback(GLFWKeyCallback.java:45)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(Native Method)
	at com.shc.silenceengine.core.glfw.GLFW3.pollEvents(GLFW3.java:114)
	at com.shc.silenceengine.core.Display.update(Display.java:385)
	at com.shc.silenceengine.graphics.GraphicsEngine.endFrame(GraphicsEngine.java:97)
	at com.shc.silenceengine.core.SilenceEngine.endFrame(SilenceEngine.java:136)
	at com.shc.silenceengine.core.Game.start(Game.java:325)
	at yirath.main.Start.main(Start.java:6)
AL lib: (EE) alc_cleanup: 1 device not closed
```

**Comment by sriharshachilakapati** [comment]
This is now fixed: See commit 6da840d
